holidays/countries/jordan.py

```diff
@@ -0,0 +1,82 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/python-holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.calendars.gregorian import GREGORIAN_CALENDAR
+from holidays.groups import InternationalHolidays, IslamicHolidays, ChristianHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Jordan(HolidayBase, IslamicHolidays, ChristianHolidays, InternationalHolidays):
+    """
+    References:
+        - https://en.wikipedia.org/wiki/Public_holidays_in_Jordan
+        - https://www.mfa.gov.jo/content/public-holidays
+    """
+
+    country = "JO"
+    default_language = "ar"
+    # %s (estimated).
+    estimated_label = tr("(تقدير) %s")
+    supported_languages = ("ar", "en_US")
+
+    def __init__(self, *args, **kwargs):
+        ChristianHolidays.__init__(self, GREGORIAN_CALENDAR)
+        InternationalHolidays.__init__(self)
+        IslamicHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # New Year's Day.
+        self._add_new_years_day(tr("رأس السنة الميلادية"))
+
+        # Independance Day.
+        self._add_holiday_may_25(tr("عيد الاستقلال"))
```

Comment:
This is more correct. It's unified across all the other AR locales
```suggestion
        self._add_holiday_may_25(tr("عيد الإستقلال"))
```